__tests__/integrations.test.js

```diff
@@ -22,10 +23,11 @@ describe('404: Incorrect URL', () => {
       .get('/api/invalid_url')
       .expect(404)
       .then(({ body }) => {
-        const expectedErrorMessage = `Oops.. We can't find that page for you`;
-        const expectedErrorParagraph = `We've searched high and low, but couldn't find the page you're looking for. How about we start from home again?`;
-        expect(body.errorMessage).toBe(expectedErrorMessage);
-        expect(body.errorParagraph).toBe(expectedErrorParagraph);
+        const expectedResponse = {
+          errorMessage: `Oops.. We can't find that page for you`,
+          errorParagraph: `We've searched high and low, but couldn't find the page you're looking for. How about we start from home again?`,
```

Comment:
this is maaaybe too much detail for a backend error, but absolutely perfect for when we go onto frontend and thinking about frontend error handling

Reply:
I had the front-end in mind when choosing the errorMessage et paragraph.